proposals/physical-cameras/README.md

```diff
@@ -314,4 +314,94 @@ class PhysicalCameraLensModelPinhole "PhysicalCameraLensModelPinhole" (
         doc = '''squeeze'''
     )
 }
+
+class PhysicalCameraExposure "PhysicalCameraExposure" (
+    doc = """
+    Describes the exposure settings common to still and cinema cameras.
+
+    These settings are used to scale the brightness of the illumination incident
+    on the camera system. The scaling factor, S, is computed as:
+       S = sensitivity_factor * iso * 2^exposure_compensation / (100 * shutterspeed * fnumber^2)
+    """
+    inherits = </APISchemaBase>
+    customData = {
+        token apiSchemaType = "singleApply"
+    }
+)
+{
+    float physcam:exposure:fnumber = 1.0 (
+        doc = """The lens aperture f-number"""
+    )
+
+    float physcam:exposure:iso = 100 (
```

Comment:
That's what the `sensitivity_factor` (open to a better name) is there for: to express the fact that there is some unknown relationship between exposure and output signal from the imaging system. The exposure equation as described in the schema doc is well defined and this fudge factor is there to account for the fact that each camera manufacturer will have a slightly different notion of what a "good exposure" is given the same incident light.

Reply:
Ah, got it. Could we call it `sensitivity_compensation`? I think `compensation` would emphasis that it's something that was actively applied, rather than something with an equation for it lying around somewhere.